### PR TITLE
Solved participants getting removed error

### PR DESCRIPTION
**Pull request by RidhamShah**

### CODE REVIEW

File: backend/packages/Upgrade/src/api/services/ExperimentService.ts
Comment (danoswaltCL) [inline backend/packages/Upgrade/src/api/services/ExperimentService.ts:647]: can we refactor into a reusable function so we are not duplicating code?

File: backend/packages/Upgrade/src/api/services/ExperimentService.ts
Comment (danoswaltCL) [inline backend/packages/Upgrade/src/api/services/ExperimentService.ts:650]: can we please name things what they are instead of using `x`? thank you

File: backend/packages/Upgrade/src/api/services/ExperimentService.ts
Comment (RidhamShah) [reply backend/packages/Upgrade/src/api/services/ExperimentService.ts]: Created a new function for both include & exclude

File: backend/packages/Upgrade/src/api/services/ExperimentService.ts
Comment (RidhamShah) [reply backend/packages/Upgrade/src/api/services/ExperimentService.ts]: Code updated

**Comment by danoswaltCL** [comment]
@ppratikcr7 @RidhamShah what does it mean when PRs are put up in draft mode for a long time? Should I wait until they are fully "opened" to leave comments / approve? I don't know if this is still being worked on.

**Review by danoswaltCL — APPROVED** [review]

**Pull request merged**
